GoogleChromeLabs/ps-analysis-tool
# Feature: Add Explorable Explanations interactive mode.

## Description
This PR aims to introduce an interactive mode where user can move the user icon with mouse and upon clicking on a circle depending on the index either auction or joinAdInterestGroup is triggered.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch.
- Go to terminal and run `cd ee && npm i && npm start`.
- Now in the browser, move the user icon to the circles and check the auction flows.

## Additional Information:



## Screenshot/Screencast

https://github.com/user-attachments/assets/46cd5c5a-d989-45c9-ad1a-f849b689d23f



---

## Checklist






Partially addresses #857
